packages/gatsby-plugin-google-tagmanager/README.md

```diff
@@ -12,14 +12,19 @@ Easily add Google Tagmanager to your Gatsby site.
 // In your gatsby-config.js
 plugins: [
   {
-    resolve: `gatsby-plugin-google-tagmanager`,
+    resolve: "gatsby-plugin-google-tagmanager",
     options: {
       id: "YOUR_GOOGLE_TAGMANAGER_ID",
 
       // Include GTM in development.
       // Defaults to false meaning GTM will only be loaded in production.
       includeInDevelopment: false,
 
+      // datalayer to be set before GTM is loaded
+      // should be a stringified object or object
```

Comment:
`generateDefaultDataLayer` checks for a function too. So not only objects are supported, right?

Reply:
thanks will have to update this comment, it indeed allows a function and an object :) stringified object isn't possible anymore.